src/EA.Weee.Integration.Tests/Handlers/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -86,7 +87,7 @@ public class WhenIGetAnAatfsSummaryData : GetAatfSummaryRequestHandlerTestBase
                     n.UpdateStatus(NoteStatus.Rejected, UserId.ToString());
                 }).Create();
 
-                request = new GetAatfSummaryRequest(aatf.Id);
+                request = new GetAatfSummaryRequest(aatf.Id, 2022);
```

Comment:
pass fixture.Create<int>() to GetAatfSummaryRequest rather than SystemTime.Now.Year. It allows us to check that the Request Compliance year is used in the tests rather than system time.